traffic_portal/app/src/modules/private/configure/deliveryServices/urlSigKeys/deliveryServiceUrlSigKeys.tpl.html

```diff
@@ -0,0 +1,55 @@
+<!--
+Licensed to the Apache Software Foundation (ASF) under one
+or more contributor license agreements.  See the NOTICE file
+distributed with this work for additional information
+regarding copyright ownership.  The ASF licenses this file
+to you under the Apache License, Version 2.0 (the
+"License"); you may not use this file except in compliance
+with the License.  You may obtain a copy of the License at
+
+  http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing,
+software distributed under the License is distributed on an
+"AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+KIND, either express or implied.  See the License for the
+specific language governing permissions and limitations
+under the License.
+-->
+
+<div id="deliveryServiceUrlSigKeysContainer">
+    <div ui-view="deliveryServiceUrlSigKeysContent">
+    	<div class="x_panel">
+		    <div class="x_title">
+		        <ol class="breadcrumb pull-left">
+		            <li><a ng-click="navigateToPath('/configure/delivery-services')">Delivery Services</a></li>
+		            <li><a ng-click="navigateToPath('/configure/delivery-services/' + deliveryService.id + '?type=' + deliveryService.type)">{{::deliveryService.displayName}}</a></li>
+		            <li class="active">URL Sig Keys</li>
+		        </ol>
+		        <div class="pull-right">
+		            <button class="btn btn-primary" title="Generate URL Sig Keys" ng-click="generateUrlSigKeys(deliveryService.xmlId)"><i class="fa">Generate URL Sig Keys</i></button>
```

Comment:
you don't need to wrap your button labels in <i class="fa">, it makes the fonts look different fyi